lib/pipe-symbol.ts

```diff
@@ -59,7 +59,7 @@ export class PipeSymbol extends Symbol {
    * @memberOf PipeSymbol
    */
   getMetadata(): Pipe | null {
-    return this.resolver.resolve(resolveForwardRef(this.symbol));
+    return this.resolver.resolve(resolveForwardRef(this.symbol as any));
```

Comment:
Why do we have to cast to `any` here?

Reply:
The following commit changed the signature of 'resolveForwardRef' between version 7.0.0 and 7.1.x. 

https://github.com/angular/angular/commit/b0476f308b5a33ee1e6479e2c602c7b01689420f

I'd actually like to propose to change the code to:
```
return this.resolver.resolve(resolveForwardRef<any>(this.symbol));
```